app/styles/app.scss

```diff
@@ -427,6 +427,66 @@ a:hover {
   font-size:12px;
 }
 
+#search-tracks {
+  text-shadow: inherit;
+}
+
+/* contains the search controls and rss feeds */
+#podcast-search-controls {
+  margin-bottom: 1.5em;
+  border: .4em dotted #5fdc82;
+}
+
+
+#rss-links {
+  display: flex;
+  justify-content: space-around;
+  padding: 0;
+}
+
+#podcast-search-controls h4 {
+  text-align: center;
+  color: white;
+}
+
+#podcast-search-controls hr {
+  border: none;
+  border-top: thick dotted #5fdc82;
```

Comment:
This looks like its the same as $green.

Reply:
I was pretty far behind master (translation conflicts) so I was using the same color code used for the podcast-search-results. Works for me after changing it to $green, thanks for the heads up.